src/common/extension.ts

```diff
@@ -0,0 +1,16 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END=
+
+export const acceptedExtension = (ext: string) =>
+    /\.epub[3]?$/.test(ext) ||
+    /\.audiobook$/.test(ext) ||
+    /\.lpf$/.test(ext) ||
+    ext === ".lcpl";
+
+export const acceptedExtensionArray = [
+    ".lcpl", ".epub", ".epub3", ".audiobook", ".lpf",
```

Comment:
Could this array be iterated in the `acceptedExtension()` function, with this check inside the loop (untested code, just for illustration):
```
if (new RegExp(`\\.${acceptedExtensionArray[i]}$`).test(ext)) { return true; }
```